modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainerV1.java

```diff
@@ -0,0 +1,165 @@
+package org.testcontainers.containers;
+
+import java.util.Collections;
+import java.util.Set;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * See <a href="https://store.docker.com/images/influxdb">https://store.docker.com/images/influxdb</a>
+ */
+public class InfluxDBContainerV1<SELF extends InfluxDBContainerV1<SELF>> extends GenericContainer<SELF> {
```

Comment:
Self typing is not needed here - we should take this as an opportunity to change it

```suggestion
public class InfluxDBContainerV1 extends GenericContainer<InfluxDBContainerV1> {
```